blacksauce/src/styles/grid/_homepageGrid.scss

```
@@ -0,0 +1,131 @@
+//background image for intro text
+.container {
+    background-image: url(../images/coverPhoto.jpg);
```

This should work without the `..`
```suggestion
    background-image: url(/images/coverPhoto.jpg);
```